dis_snek/models/discord/channel.py

```diff
@@ -845,6 +846,61 @@ def permissions_for(self, instance: Snowflake_Type) -> Permissions:
 
             return self.permissions_for(instance)
 
+    async def add_permission(
+        self,
+        target: Union["PermissionOverwrite", "models.Role", "models.User"],
+        type: "OverwriteTypes",
+        allow: Optional[List["Permissions"]] = None,
+        deny: Optional[List["Permissions"]] = None,
```

Comment:
I think should have support for a single `int` value passed in.

Reply:
Added support for ints as well as a list of Permissions